include/server/api/base_parameters_grammar.hpp

```diff
@@ -72,7 +72,7 @@ struct BaseParametersGrammar : boost::spirit::qi::grammar<std::string::iterator>
 
         alpha_numeral = +qi::char_("a-zA-Z0-9");
         polyline_chars = qi::char_("a-zA-Z0-9_.--[]{}@?|\\%~`^");
-        base64_char = qi::char_("a-zA-Z0-9--_");
+        base64_char = qi::char_("a-zA-Z0-9--_=");
```

Comment:
Because in boost spirit `-` doesn't mean what you think it means. This is escaping for `-`.